### PR TITLE
successfully created the getEvent controller in eventController.js

### PR DESCRIPTION
**Pull request by kennie-larkson**
The task to Create a new endpoint for getting single event has been created and tested to be working as expected.
So go ahead and shoot your shot at that endpoint and I bet it would respond with a 200 lol

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DATABASE_URL = mongodb+srv://lawallarkson:$vMv8a3Di9tpSQY@halaqaat.sd2ol.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
```
Comment (Yusadolat) [inline .env:1]: You still commit your .env file lol

Comment (Yusadolat) [reply .env:1]: Make sure your .gitignore file includes ``.env/``

**Review by Yusadolat — COMMENTED** [review]
Another observation is the Pagination you worked on yesterday isn't reflecting here. Merging this will cause a conflict as the existing code has an updated code this doesn't have.

File: .env (hunk 1 above)
Comment (kennie-larkson) [reply .env]: All sorted Chief

**Pull request merged**